.travis.yml

```diff
@@ -1,24 +1,29 @@
 language: python
+
```

Comment:
Is there any reason to keep Travis? I think it might be a good idea to switch to GitHub Actions.

They are way more reliable. And the transition is quite simple. 🙂 

Many other `aio-libs` are already using it.

Probably, the only problem is the release part, since someone with the root access must do it.

Reply:
Ok. I will try to replace Travis with GitHub Actions.

"Release part" already implemented as GitHub Actions.